server/models/views/target-area.js

```diff
@@ -87,4 +87,8 @@ class ViewModel {
   }
 }
 
+function messageValidator (message) {
+  const strippedMessage = message.replace(/(\r?\n)+/g, '\n')
+  return strippedMessage.split('\n').map(p => `<p>${p}</p>`).join(' ')
```

Comment:
There is an argument to say that we shouldn't be adding html tags here we should just return an array and iterate the array in the template to generate the paragraphs.

Reply:
I'm guessing the styling for these paragraphs will be through the class of the containing div (something like #parentclass.p). An example would be to finesse the spacing between the paragraphs rather than leave it to the default.

I mention this as it is something we did for the FGS.

Reply:
I do agree with this however throughout the whole target area class we seem to be creating html and passing it to the view. Argument for an an all or nothing approach - lines 21-26 for example doing the same.